README.md

```diff
@@ -1,2 +1,22 @@
-# openshift-apim
```

Comment:
Why this line is removed?

Reply:
Rewrite the README file to tally with helm readme file